common/src/main/java/org/conscrypt/NativeCrypto.java

```diff
@@ -46,12 +46,8 @@
 
 /**
  * Provides the Java side of our JNI glue for OpenSSL.
- *
- * @hide
  */
-@Internal
-public final class NativeCrypto {
-
+final class NativeCrypto {
```

Comment:
This will need to be changed back to public for now for Android. I should be able to fix this next week.